datafiles/templates/upload.html.st

```diff
@@ -93,6 +93,22 @@ service.</p>
 material that you upload. For example, check that your tarball includes the
 license files of any 3rd party code that you include.</p>
 
+<h3>Public repositories</h3>
+
+It is preferable if any web pages linked from the following fields are publicly accessible:
+<ul>
+<li> Bug report URL
```

Comment:
I suspect: missing `</li>`.

Reply:
Fixed, thanks. (Too used to markdown these days.)